src/v1/components/surfaces/FixedPanel/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { HTMLAttributes } from "react";
```

Comment:
**Requested change**
I believe [code-convention](https://github.com/Telicent-io/docs/blob/main/fe/guild/decisions/file-folder-structure.md) is `FixedPanel/FixedPanel.tsx`

sidenote: And I don't know if we documented it, but I _think_ our emerged convention is to only use `index.tsx` as [barrel-files](https://www.google.com/search?q=barrel+files) for the purposes of exporting